<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-077G3EM" name="com.RingPayScripts.PromoCode_Scripts" tests="29" failures="3" timestamp="2022-12-14T11:14:19 IST" time="250.411" errors="5">
  <testcase name="userDetailsAPI_PromoCodeJourney" time="8.387" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="onloadAPI_PromoCodeJourney" time="0.589" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="sendOtp_Positive_PromoCodeJourney" time="3.216" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="userToken_PromoCodeJourney" time="8.430" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="promoCode_PromoCodeJourney" time="3.082" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="updateUserDetils_200_PromoCodeJourney" time="25.977" classname="com.RingPayScripts.PromoCode_Scripts">
    <failure type="java.lang.AssertionError" message="expected object to not be null">
      <![CDATA[java.lang.AssertionError: expected object to not be null
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertNotNull(Assert.java:874)
at org.testng.Assert.assertNotNull(Assert.java:858)
at com.utility.Validation.assertRequest_IdNotNullBodyValidation(Validation.java:237)
at com.business.RingPay_PromoCode_Journey.RegisterUser_UserAuthenticate.userToken_Positive(RegisterUser_UserAuthenticate.java:49)
at com.business.RingPay_PromoCode_Journey.RegisterUser_UpdateUser.updateUser_Positive(RegisterUser_UpdateUser.java:36)
at com.RingPayScripts.PromoCode_Scripts.updateUserDetils_200_PromoCodeJourney(PromoCode_Scripts.java:142)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- updateUserDetils_200_PromoCodeJourney -->
  <system-out/>
  <testcase name="loginUser_PromoCodeJourney" time="3.191" classname="com.RingPayScripts.PromoCode_Scripts">
    <failure type="java.lang.AssertionError" message="expected object to not be null">
      <![CDATA[java.lang.AssertionError: expected object to not be null
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertNotNull(Assert.java:874)
at org.testng.Assert.assertNotNull(Assert.java:858)
at com.utility.Validation.assertRequest_IdNotNullBodyValidation(Validation.java:237)
at com.business.RingPay_PromoCode_Journey.RegisterUser_Login.login_Positive(RegisterUser_Login.java:54)
at com.RingPayScripts.PromoCode_Scripts.loginUser_PromoCodeJourney(PromoCode_Scripts.java:173)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginUser_PromoCodeJourney -->
  <system-out/>
  <testcase name="registerUser_PromoCodeJourney" time="3.119" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="locationRequire_PromoCodeJourney" time="2.985" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="getUserDetails_PromoCodeJourney" time="2.724" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="user_onboarding_PromoCodeJourney" time="39.073" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="create_bnpl_transaction_PromoCodeJourney" time="17.965" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="upadate_user_status_PromoCodeJourney" time="0.992" classname="com.RingPayScripts.PromoCode_Scripts">
    <failure type="java.lang.AssertionError" message="expected object to not be null">
      <![CDATA[java.lang.AssertionError: expected object to not be null
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertNotNull(Assert.java:874)
at org.testng.Assert.assertNotNull(Assert.java:858)
at com.utility.Validation.assertRequest_IdNotNullBodyValidation(Validation.java:237)
at com.business.RingPay_MerchantQRCode_Journey.RegisterUser_UserAuthenticate.userToken_Positive(RegisterUser_UserAuthenticate.java:49)
at com.utility.Utilities.UpdateUserStatusAPI(Utilities.java:1667)
at com.business.RingPay_PromoCode_Journey.BasicDetailScreen_Update_User_Status.updateUserStatus_Positive(BasicDetailScreen_Update_User_Status.java:32)
at com.RingPayScripts.PromoCode_Scripts.upadate_user_status_PromoCodeJourney(PromoCode_Scripts.java:277)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- upadate_user_status_PromoCodeJourney -->
  <system-out/>
  <testcase name="check_Application_Eligibility_PromoCodeJourney" time="12.407" classname="com.RingPayScripts.PromoCode_Scripts">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.utility.Validation.assertTrue(Validation.java:342)
at com.business.RingPay_PromoCode_Journey.BasicDetailScreen_Check_Application_Eligibility.checkApplicationEligibility_Positive(BasicDetailScreen_Check_Application_Eligibility.java:46)
at com.RingPayScripts.PromoCode_Scripts.check_Application_Eligibility_PromoCodeJourney(PromoCode_Scripts.java:290)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- check_Application_Eligibility_PromoCodeJourney -->
  <system-out/>
  <testcase name="get_Offer_Details_PromoCodeJourney" time="9.957" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="accept_Offer_PromoCodeJourney" time="9.367" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="offerDetailsScreen_CheckApplicationEligibility_PromoCodeJourney" time="12.855" classname="com.RingPayScripts.PromoCode_Scripts">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.utility.Validation.assertTrue(Validation.java:342)
at com.business.RingPay_PromoCode_Journey.OfferDetailsScreen_Check_Application_Eligibility.OfferDetailsScreen_CheckApplicationEligibility_Positive(OfferDetailsScreen_Check_Application_Eligibility.java:34)
at com.RingPayScripts.PromoCode_Scripts.offerDetailsScreen_CheckApplicationEligibility_PromoCodeJourney(PromoCode_Scripts.java:331)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- offerDetailsScreen_CheckApplicationEligibility_PromoCodeJourney -->
  <system-out/>
  <testcase name="getPinDetails" time="3.429" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="sendOtpForPin" time="2.903" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="setResetPin" time="5.710" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="afterResetPin_getPinDetails" time="2.720" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="bnplLines_PromoCodeJourney" time="2.740" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="paymentOption_PromoCodeJourney" time="22.651" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="transaction_Initiate_PromoCodeJourney" time="27.572" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="transactionComplete_PromoCodeJourney" time="3.752" classname="com.RingPayScripts.PromoCode_Scripts">
    <error type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.business.RingPay_PromoCode_Journey.Bnpl_Txn_Transaction_Complete.transactionComplete(Bnpl_Txn_Transaction_Complete.java:34)
at com.RingPayScripts.PromoCode_Scripts.transactionComplete_PromoCodeJourney(PromoCode_Scripts.java:464)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 33 more
]]>
    </error>
  </testcase> <!-- transactionComplete_PromoCodeJourney -->
  <system-out/>
  <testcase name="homeScreenForCurrentSpends_PromoCodeJourney" time="4.018" classname="com.RingPayScripts.PromoCode_Scripts">
    <error type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.business.RingPay_PromoCode_Journey.Repayment_Home_Screen_For_Current_Spends.current_Spent(Repayment_Home_Screen_For_Current_Spends.java:37)
at com.RingPayScripts.PromoCode_Scripts.homeScreenForCurrentSpends_PromoCodeJourney(PromoCode_Scripts.java:475)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 33 more
]]>
    </error>
  </testcase> <!-- homeScreenForCurrentSpends_PromoCodeJourney -->
  <system-out/>
  <testcase name="paymentValidate_PromoCodeJourney" time="2.764" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="notifyPaymentDone_PromoCodeJourney" time="2.572" classname="com.RingPayScripts.PromoCode_Scripts"/>
  <system-out/>
  <testcase name="getSettlementStatus_PromoCodeJourney" time="5.264" classname="com.RingPayScripts.PromoCode_Scripts">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.business.RingPay_PromoCode_Journey.TransactionDetails_Get_Settlement_Status.getSettlementStatus(TransactionDetails_Get_Settlement_Status.java:29)
at com.RingPayScripts.PromoCode_Scripts.getSettlementStatus_PromoCodeJourney(PromoCode_Scripts.java:507)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- getSettlementStatus_PromoCodeJourney -->
  <system-out/>
</testsuite> <!-- com.RingPayScripts.PromoCode_Scripts -->
